### PR TITLE
Add service example

### PR DESCRIPTION
**Pull request by dnoegel**

### CODE REVIEW

File: exampleplugins/Frontend/SwagService/Bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+ */
+class Shopware_Plugins_Frontend_SwagService_Bootstrap extends Shopware_Components_Plugin_Bootstrap
+{
+    /**
+     * Register the namespace of our plugin globally
+     */
+    public function afterInit()
+    {
+        $this->Application()->Loader()->registerNamespace(
+            'Shopware\SwagService',
```
Comment (bcremer) [inline exampleplugins/Frontend/SwagService/Bootstrap.php]: Can you `ShopwarePlugins\SwagService` so it is consistent with the other example plugins.

File: exampleplugins/Frontend/SwagService/Bootstrap.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    /**
+     * Return an instance of our new service class
+     *
+     * @return \Shopware\SwagService\Component\TaxCalculator
+     */
+    public function onInitTaxCalculator()
+    {
+        return new \Shopware\SwagService\Component\TaxCalculator(
+            $this->get('corelogger')
```
Comment (bcremer) [inline exampleplugins/Frontend/SwagService/Bootstrap.php]: We should use `pluginlogger` here. The `corelogger` should be reserved for logging within the shopware core.

**Comment by tiagojsag** [comment]
I just fixed a couple of minor typos, nothing special. @dnoegel you will be a native speaker soon (so you can stop using that german language thing) :P

**Comment by dnoegel** [comment]
Thanks @tiagojsag for your fixes. @bcremer: Changed plugin namespace

**Pull request merged**